tests/test_stream_connect.podman.cil

```diff
@@ -0,0 +1,7 @@
+(block my_container
+    (blockinherit container)
+    (allow process network_container.process ( unix_stream_socket ( connectto ))) 
+    (allow process network_container.socket ( sock_file ( getattr write open append ))) 
+    (allow process process ( capability ( chown dac_override fsetid fowner mknod net_raw setgid setuid setfcap setpcap net_bind_service sys_chroot kill audit_write ))) 
```

Comment:
Why does a process need all of these capabilities.

Reply:
These capabilities are included in podman inspection json when you execute: # podman run -it fedora bash

Reply:
Ok, but they are not related to stream_connect.

Reply:
Yeas, but this file is part of our CI. It tests if udica generates policies correctly.

Reply:
Ok.  I though it was a template file.  Stupidly, I did not pay attention to the tests directory.

Reply:
np Dan.